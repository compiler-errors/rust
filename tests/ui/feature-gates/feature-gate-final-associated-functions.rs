trait Foo {
    final fn bar() {}
    //~^ ERROR `final` on trait functions is experimental
}

fn main() {}
