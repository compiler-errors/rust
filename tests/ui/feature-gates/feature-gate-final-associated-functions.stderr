error[E0658]: `final` on trait functions is experimental
  --> $DIR/feature-gate-final-associated-functions.rs:2:5
   |
LL |     final fn bar() {}
   |     ^^^^^
   |
   = note: see issue #1 <https://github.com/rust-lang/rust/issues/1> for more information
   = help: add `#![feature(final_associated_functions)]` to the crate attributes to enable
   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0658`.
